frontend/src/components/AlertRuleDialog/components/AlertRuleDialogButtons.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import { Box } from '@mui/material';
+
+import { StyledButton } from '../AlertRuleDialog.styles';
+
+import { AlertRuleStepBaseProps } from '../AlertRuleDialog.type';
+import { constants } from '../alertRuleDialog.constants';
+
+const {
+  buttons: { back, next }
+} = constants;
+
+interface AlertRuleDialogButtonsProps extends AlertRuleStepBaseProps {
+  disabled: boolean;
+}
+
+export const AlertRuleDialogButtons = ({
+  activeStep,
+  handleBack,
+  handleNext,
+  disabled
+}: AlertRuleDialogButtonsProps) => (
+  <Box marginTop="50px" marginLeft="auto">
+    {activeStep !== 0 && (
```

Comment:
Don't use logic inside UI, put in a const

Reply:
I don't see much point in creating a variable when the logic is not complex or only used in one place.